index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mobile</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="intro">
+        <header id="header">
+            <h3 id="logo" class="content-padding"><a href="#intro">My Logo</a></h3>
+            <nav id="navbar">
+                <ul>
+                    <li class="content-padding"><a href="#">Portfolio</a></li>
+                    <li class="content-padding"><a href="#">About</a></li>
+                    <li class="content-padding"><a href="#">Contact</a></li>
+                </ul>
+                <a href="#" class="content-padding" id="menu"><img src="images/menu-icon.png" alt="menu"></a>
+            </nav>
+
+        </header>
+        <section id="headline">
+            <h1>I’m Alexandra <br>Glad to see you!</h1>
```

Comment:
Kindly note that the `Glad to see you!` text should be a`<hx>` e.g `<h2>` and styled using flexbox to match the design. This way will be semantically correct and the screen readers and other technologies will thank you.